trunion/crypto.py

```diff
@@ -17,7 +17,8 @@
 import random
 import re
 
-CERTIFICATE_RE = re.compile(r"-----BEGIN CERTIFICATE-----.+?-----END CERTIFICATE-----", re.S)
+CERTIFICATE_RE = re.compile(r"-----BEGIN CERTIFICATE-----.+"
+                            "-----END CERTIFICATE-----", re.S)
```

Comment:
Does the second line need to be a raw string as well so that string coercion works right?